package com.momenta.repository;

import com.momenta.model.Redemption;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;

public interface RedemptionRepository extends JpaRepository<Redemption, Long> {
    Optional<Redemption> findByCode(String code);
}
